packages/web/src/components/artist/ArtistCard.tsx

```diff
@@ -0,0 +1,163 @@
+import React from 'react'
+
+import { connect } from 'react-redux'
+import { Dispatch } from 'redux'
+
+import { ID } from 'common/models/Identifiers'
+import { CoverPhotoSizes, ProfilePictureSizes } from 'common/models/ImageSizes'
+import { Supporting } from 'common/models/Tipping'
+import FollowButton from 'components/follow-button/FollowButton'
+import Stats, { StatProps } from 'components/stats/Stats'
+import {
+  setUsers,
+  setVisibility
+} from 'store/application/ui/userListModal/slice'
+import {
+  UserListEntityType,
+  UserListType
+} from 'store/application/ui/userListModal/types'
+
+import styles from './ArtistCard.module.css'
+import { ArtistCover } from './ArtistCover'
+import { ArtistSupporting } from './ArtistSupporting'
+
+type ArtistCardProps = {
+  description: string
+  trackCount: number
+  playlistCount: number
+  followerCount: number
+  followingCount: number
+  doesFollowCurrentUser: boolean
+  userId: number
+  name: string
+  handle: string
+  profilePictureSizes: ProfilePictureSizes
+  coverPhotoSizes: CoverPhotoSizes
+  isArtist: boolean
+  onNameClick: () => void
+  following: boolean
+  onFollow: () => void
+  onUnfollow: () => void
+  supportingList: Supporting[]
+  onSupportingClick: () => void
+  setUsers: (id: ID) => void
+  openModal: () => void
+} & ReturnType<typeof mapDispatchToProps>
+
+const ArtistCard = ({
```

Comment:
how about we add a comment describing what this component is – ArtistCard feels pretty vague to me

Reply:
😆 I just saw this was a port from JS. nevermind